src/Modal/ModalContents.js

```diff
@@ -63,8 +64,22 @@ export default class ModalContents extends Util.mixin(BindMixin) {
   }
 
   checkHeight() {
+    let prevContentHeight = this.heightInfo.innerContentHeight;
```

Comment:
Could we get some tests in for this function?
I wanna make sure that we are doing it right with these extra calculations